x-pack/plugins/actions/server/builtin_action_types/es_index.ts

```diff
@@ -85,9 +85,19 @@ async function executor(
     refresh: config.refresh,
   };
 
-  let result;
   try {
-    result = await services.callCluster('bulk', bulkParams);
+    const result = await services.callCluster('bulk', bulkParams);
+
+    const err = find(result.items, 'index.error.reason');
+    if (err) {
+      throw new Error(
```

Comment:
> BTW, curious how you found this - seems like a general design bug with this action (not checking the bulk results for individual errors), don't remember seeing an issue for it tho.

Haha because it didn't fail! We swallowed the error and told the user it passed...

Reply:
> Is there a reason we're `throw`ing here instead of passing back a `{ status: 'error', ... }` object? I'm sure it will be handled fine, but I guess I tend to want to have action executors try to catch "common" errors and return the error status object - some day we may want to reserve catching errors in the caller as some kind of "misbehaving action executor" or such ...

Yeah, I considered that and decided to only have one `error` handling block, but happy to change that 👍